js/components/controlBar.js

```diff
@@ -63,6 +63,22 @@ var ControlBar = React.createClass({
     this.props.controller.toggleFullscreen();
   },
 
+  handleLiveClick: function(evt) {
+    evt.stopPropagation();
+    evt.cancelBubble = true;
+    evt.preventDefault();
+    this.props.controller.seek(this.props.duration);
+  },
+
+  checkLiveCursor: function() {
+    var timeShift = this.props.currentPlayhead - this.props.duration;
+    if (Math.abs(timeShift) < 2) {
+      ReactDOM.findDOMNode(this.refs.LiveButton).style.cursor="default";
```

Comment:
Can you please do it?  I won't be available for a while.  Thanks!

On Friday, May 13, 2016, Xenia Serbin <notifications@github.com> wrote:

> In js/components/controlBar.js
> <https://github.com/ooyala/html5-skin/pull/525#discussion_r63223605>:
>
> > @@ -63,6 +63,22 @@ var ControlBar = React.createClass({
> >      this.props.controller.toggleFullscreen();
> >    },
> >
> > +  handleLiveClick: function(evt) {
> > +    evt.stopPropagation();
> > +    evt.cancelBubble = true;
> > +    evt.preventDefault();
> > +    this.props.controller.seek(this.props.duration);
> > +  },
> > +
> > +  checkLiveCursor: function() {
> > +    var timeShift = this.props.currentPlayhead - this.props.duration;
> > +    if (Math.abs(timeShift) < 2) {
> > +      ReactDOM.findDOMNode(this.refs.LiveButton).style.cursor="default";
>
> We are trying not to change styling outside of css when possible. Here,
> for example, instead of changing the style you could change the class name
> for the element (using ClassNames, see volume as an example) depending on
> its status. You can do it in populateControlBar, and remove onMouseOver
> altogether.
>
> var liveClass = ClassNames({
>     "oo-control-bar-item oo-live oo-live-indicator": true,
>     "oo-live-nonclickable": Math.abs(this.props.currentPlayhead - this.props.duration) < 2
>   });
>
> and then set 'cursor: default' in css for oo-live-nonclickable
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-skin/pull/525/files/36666e08f032caab3a52688a50887e1ebf81eabb#r63223605>
>


-- 
-- 
Dimitry Nasledov
dima@ooyala.com